capa/rules/__init__.py

```diff
@@ -1618,80 +1806,235 @@ def filter_rules_by_meta(self, tag: str) -> "RuleSet":
                             break
         return RuleSet(list(rules_filtered))
 
-    def match(self, scope: Scope, features: FeatureSet, addr: Address) -> Tuple[FeatureSet, ceng.MatchResults]:
+    # unstable
```

Comment:
Does this need to be resolved before release?

Reply:
and/or do we need TODOs around these?

Reply:
I've been using the term "unstable" to indicate that library users should not rely on the function/method/datatype/etc. I guess like "private" but more along the lines of "not sure that this has settled yet". This way we can do further experimentation without waiting for a major release. 

Open to discussion, other terms, or just adding more comments to explain this in code.

Reply:
Thanks for all the updates!